### ISSUE TITLE
The icon only shows on the last mortified file.

### ISSUE THREAD

**Issue by weizhou-chaojixx** [Xuyuanp/nerdtree-git-plugin#150]
#### Self-Diagnosis

- [* ] I have searched the [issues](https://github.com/Xuyuanp/nerdtree-git-plugin/issues) for an answer to my question.
- [* ] I have reviewed the NERDTree documentation(README.md).
- [* ] I have searched the web for an answer to my question.

#### Environment (for bug reports)
- [ ] Operating System: Ubuntu18.04
- [ ] Vim/Neovim version `:echo v:version`: 800
- [ ] NERDTree version, found on 1st line in NERDTree quickhelp `?`: 
- [ ] vimrc settings
    ```vim
    " all settings about nerdtree and other plugins
     I didn't change anything
    - Other NERDTree-dependent Plugins
        - [ ] jistr/vim-nerdtree-tabs
        - [ *] ryanoasis/vim-devicons
        - [ *] tiagofumo/vim-nerdtree-syntax-highlight
        - [ *] Others (specify): vim-signify 
    - [ *] I've verified the issue occurs with only nerdtree-git-plugin installed.

#### Steps to Reproduce the Issue
1. When I modify multiple files in one repo, only the last modified file shows the icon, the icon of the previous modified file automatically disappear.

#### Current Result (Include screenshots where appropriate.)
![image](https://user-images.githubusercontent.com/6397749/90350431-264c0600-e00b-11ea-886a-e14c36a11a2a.png)

#### Expected Result
![image](https://user-images.githubusercontent.com/6397749/90350446-319f3180-e00b-11ea-89e9-5b1e9765725a.png)

**Comment by Xuyuanp** [comment]
Does force refreshing nerdtree fix the icons?

**Comment by weizhou-chaojixx** [comment]
Nope, only the lastest modified file shows the icon.

**Comment by Xuyuanp** [comment]
Can you switch to the `doctor` branch, and execute `call gitstatus#doctor#Say()` after nerdtree opened?

**Comment by weizhou-chaojixx** [comment]
Sure, the results are shown in the following screenshot, you can see many files have been modified via original git status. However, only one file is listed in "NerdtreeGitStatusCashe". Could you identify the problem?
 
![image](https://user-images.githubusercontent.com/6397749/90697059-8a5d0d00-e24b-11ea-82de-441b180308e3.png)
![image](https://user-images.githubusercontent.com/6397749/90697067-8e892a80-e24b-11ea-9195-6a1ff831d8f2.png)
![image](https://user-images.githubusercontent.com/6397749/90697081-96e16580-e24b-11ea-8c0d-c2b69ebcb4b1.png)
![image](https://user-images.githubusercontent.com/6397749/90697096-9c3eb000-e24b-11ea-9e05-873db0d4c9d8.png)

**Comment by Xuyuanp** [comment]
Confirmed.

The job didn't flush buffers in `raw` mode when closed. This problem exists for vim older than v8.1.0870.

> Patch 8.1.0870
Problem:    Vim doesn't use the new ConPTY support in Windows 10.
Solution:   Use ConPTY support, if available. (Nobuhiro Takasaki, closes #3794)
Files:      runtime/doc/eval.txt, runtime/doc/options.txt,
            runtime/doc/terminal.txt, src/channel.c, src/evalfunc.c,
            src/globals.h, src/option.c, src/option.h, src/os_win32.c,
            src/proto/terminal.pro, src/structs.h, src/terminal.c,
            src/testdir/gen_opt_test.vim, src/testdir/test_autocmd.vim,
            src/testdir/test_mksession.vim, src/testdir/test_terminal.vim

`:h patches-8.2`

It looks like the changes in file [`src/channel.c`](https://github.com/vim/vim/commit/aa5df7e3127dff6b7336df0903f5c569a40eb174#diff-3eef145ca58a199bac70eef155e806ae)  fixed this issue.

A temporary solution is to change lines 92~93 of file `autoload/gitstatus/job.vim`
from
```vim
        let options = {
                    \ 'out_cb':   { _ch, data  -> self.onStdoutCB([data]) },
                    \ 'err_cb':   { _ch, data  -> self.onStderrCB([data]) },
                    \ 'exit_cb':  { _ch, _data -> self.onExitCB() },
                    \ 'out_mode': 'raw',
                    \ 'err_mode': 'raw',
                    \ }
```
to
```vim
        let options = {
                    \ 'out_cb':   { _ch, data  -> self.onStdoutCB([data]) },
                    \ 'err_cb':   { _ch, data  -> self.onStderrCB([data]) },
                    \ 'exit_cb':  { _ch, _data -> self.onExitCB() },
                    \ 'out_mode': 'nl',
                    \ 'err_mode': 'nl',
                    \ }
```

I'm going to do more tests.
